prisma/schema.prisma

```diff
@@ -205,16 +219,16 @@ enum RoleDescription {
   FRONTEND
   BACKEND 
   FULLSTACK
-  MANAGER
-  DIRECTOR
-  VPE
   MACHINE_LEARNING
   DATA_SCIENCE
   INFRASTRUCTURE
   SRE
   DEVOPS
   QA
+  SECURITY
   QA_MANAGER
+  DIRECTOR
+  VPE
```

Comment:
What's VPE? Isn't it just VP? Or does 'E' stand for Executive? In which case, should this just be `Executive`?

Reply:
VPE stands for VP of Engineering. If it's not explicit, we could change to the long name.